init-tools.cmd

```diff
@@ -68,4 +70,6 @@ if not [%INIT_TOOLS_ERRORLEVEL%]==[0] (
 :: Create sempahore file
 echo Done initializing tools.
 echo Init-Tools.cmd completed for BuildTools Version: %BUILDTOOLS_VERSION% > "%BUILD_TOOLS_SEMAPHORE%"
+
+type %INIT_TOOLS_LOG%
```

Comment:
Yes I'm just using it to debug what is taking some of the legs so long to init tools.